management/dbaas/xtra_db_cluster_test.go

```diff
@@ -138,7 +138,7 @@ func TestXtraDBClusterServer(t *testing.T) {
 		}
 
 		_, err = dbaasClient.Default.XtraDBCluster.UpdateXtraDBCluster(&paramsUpdatePXC)
-		pmmapitests.AssertAPIErrorf(t, err, 400, codes.InvalidArgument, `invalid field Params.Pxc.DiskSize: value '0' must be greater than '0'`)
+		pmmapitests.AssertAPIErrorf(t, err, 500, codes.InvalidArgument, `XtraDB cluster is not ready`)
```

Comment:
it's weird that we return different status codes and messages for the same error for xtradb and psmdb. Can we fix on dbaas-controller side?

Reply:
Yes, it is unified on dbaas-controller side.